diff/deleter.go

```diff
@@ -130,11 +129,6 @@ func (d *Deleter) deleteRelation(id int64, deleteRefs bool, deleteMembers bool)
 		if m.Way == nil {
 			continue
 		}
-		err := d.osmCache.Coords.FillWay(m.Way)
```

Comment:
@olt hWhat is the purpose of `d.osmCache.Coords.FillWay(m.Way)`?
This branch was never activated because the Expireor was always `nil` but now causes the diff process
to fail since there is an expireor. I guess that is old code we can remove?